### PR TITLE
Add Feature for Verse Breaks

### PR DESCRIPTION
**Pull request by BryanHo10**
Minor Pull Request for User Story:

Requirement: Provide option to separate each verse onto a new line.

### CODE REVIEW

File: USFMToolsSharp/HtmlRenderer.cs [hunk 1]
```diff
@@ -137,6 +137,12 @@ private string RenderMarker(Marker input)
                         output.Append(RenderMarker(marker));
                     }
                     output.AppendLine($"</span>");
+
+                    // New Line after each Verse
+                    if (ConfigurationHTML.separateVerses)
+                    {
+                        output.AppendLine("<br class=\"versebreak\"/>");
```
Comment (rbnswartz) [inline USFMToolsSharp/HtmlRenderer.cs]: I don't think you need the css class.

Comment (BryanHo10) [reply USFMToolsSharp/HtmlRenderer.cs]: Yeah you're right. It isn't considered good practice attach styling to the <br> tag.

Comment (rbnswartz) [reply USFMToolsSharp/HtmlRenderer.cs]: @BryanHo10 So we should probably remove it?

Comment (BryanHo10) [reply USFMToolsSharp/HtmlRenderer.cs]: Oops, sorry. Forgot to push it.

**Pull request merged**